src/test/java/org/assertj/core/internal/maps/Maps_assertDoesNotContainKeys_Test.java

```diff
@@ -71,7 +78,7 @@ void should_fail_if_actual_contains_key() {
     Throwable error = catchThrowable(() -> maps.assertDoesNotContainKeys(info, actual, new String[] { key }));
```

Comment:
use `expectAssertionError` to remove `assertThat(error).isInstanceOf(AssertionError.class);` and `error` local variable

Reply:
not sure why you resolved this one @scordio ;-)

Reply:
Becase I rewrote the entire `Maps_assertDoesNotContainKeys_Test` 🙂